eZ/Publish/Core/Persistence/Legacy/Content/Type/Handler.php

```diff
@@ -619,4 +619,23 @@ public function publish( $contentTypeId )
 
         $this->updateHandler->publishNewType( $toType, Type::STATUS_DEFINED );
     }
+
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\Type\Handler::getFieldMap
+     */
+    public function getFieldMap()
+    {
+        $fieldMap = [];
```

Comment:
Yes, thanks for the heads up.

> each with the minimal php version configured, we should be able to benefit from it.

I had a bug from that on 5.3 recently (closure), but it didn't really go through PHPStorm as cherry-picks were clean. I think most often that will be the case. We would (I would at least) benefit more from the automatic selection of PHP version to run the tests, but that seems like a bit harder to achieve :)

> It also highlights that backports should trigger travis tests :-)

You mean Shippable?